src/StopButton.scss

```diff
@@ -1,21 +1,41 @@
 @import 'variables';
 
-.StopButton-svg {
-    width: 2.6rem;
-    height: 2.6rem;
+.StopButton {
+    height: 2rem;
+    width: 2rem;
+    font-size: 1.2rem;
+
+    box-shadow: 0 0 0 0.2rem black;
```

Comment:
We need to update any usage of "black" in StopButton to be "#1E1E1E" to match designs in Figma.